libvast/src/defaults.cpp

```diff
@@ -22,16 +22,6 @@
 
 namespace vast::defaults::import {
 
-caf::atom_value table_slice_type(const caf::actor_system& sys,
-                                 const caf::settings& options) {
-  // The parameter import.table-slice-type overrides system.table-slice-type.
-  if (auto val = caf::get_if<caf::atom_value>(&options,
-                                              "import.table-slice-type"))
-    return *val;
-  return get_or(sys.config(), "system.table-slice-type",
-                system::table_slice_type);
-}
```

Comment:
If I remember correctly, this function is used so that the "default" table slice type can be changed to `msgpack` in vast-pro. Unless I'm missing something, removing it will change this behaviour?

Reply:
I'll take a look at this one specifically with VAST Pro in mind and will report back.

Reply:
That's not an issue, here's the relevant code snippet:

```cpp
struct pro_configuration : default_configuration {
  pro_configuration() {
    // Adjust defaults.
    put(content, "import.table-slice-type", caf::atom("msgpack"));
  }
};
```